src/store/Workspaces/index.ts

```diff
@@ -209,24 +209,7 @@ export const actionCreators: ActionCreators = {
       }
     } catch (e) {
       dispatch({ type: 'RECEIVE_ERROR' });
-
-      const errorMessage = e?.message || '';
```

Comment:
it's already a part of some helper method we have, right?

Reply:
that's correct

Reply:
Our [helper](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/helpers/getErrorMes) doesn't cover the next line:
```
 const responseMessage = e?.response?.data?.message || '';
 ```
 @akurinnoy 
 Do we need to add this structure to our helper?

Reply:
that makes sense, I'm preparing update

Reply:
Thanks